nixos/modules/tasks/network-interfaces-scripted.nix

```diff
@@ -263,6 +268,10 @@ let
               ip link del "${n}" || true
               rm -f /run/${n}.interfaces
             '';
+            preStop = ''
+              # Save currently attached slaves
+              ip link | grep master | awk '{print $2,$9}' | grep "${n}" | cut -d ":" -f 1 > /run/${n}.slaves
```

Comment:
I know there is a webpage called useless-use-of-cat. But this here is a useless use of awk and double grep.

Why not just do `ip link |grep "master ${n}" |cut -d' ' f 9` and start two processes less?